### PR TITLE
Fixed bug 1267409 - updated Gigabit Twitter handle

### PR DESCRIPTION
**Pull request by mmmavis**
Fixed bugzilla bug 1267409
https://bugzilla.mozilla.org/show_bug.cgi?id=1267409 :smile:

### CODE REVIEW

File: bedrock/gigabit/templates/gigabit/apply.html [hunk 1]
```diff
@@ -132,7 +132,7 @@ <h1 class="large center">{{ _('Apply') }}</h1>
         <div class="content">
           <h2>{{ _('How to Apply') }}</h2>
           <p>
-          {% trans url_mailto='mailto:gigabit@mozilla.com', url_twitter='//twitter.com/mozillagigabit', url_hive_kc='http://hivekc.org', url_hive_cha='http://hivecha.org' %}
+          {% trans url_mailto='mailto:gigabit@mozilla.com', url_twitter='//twitter.com/MozGig', url_hive_kc='http://hivekc.org', url_hive_cha='http://hivecha.org' %}
```
Comment (craigcook) [inline bedrock/gigabit/templates/gigabit/apply.html]: Let's take this opportunity to make the Twitter URL explicitly `https://`.

File: bedrock/gigabit/templates/gigabit/gigabit.html [hunk 2]
```diff
@@ -134,7 +134,7 @@ <h1 class="large">{{ _('Mozilla Gigabit Community Fund') }}</h1>
             <h2>{{ _('Get in Touch') }}</h2>
             <p>
               {% trans %}
-              Find us on <a href="http://twitter.com/mozillagigabit">Twitter</a> or send us an <a href="mailto:gigabit@mozilla.com">email</a>.
+              Find us on <a href="http://twitter.com/MozGig">Twitter</a> or send us an <a href="mailto:gigabit@mozilla.com">email</a>.
```
Comment (craigcook) [inline bedrock/gigabit/templates/gigabit/gigabit.html]: Ideally these URLs shouldn't be part of the string, but I also don't think this pages is localized so this string change shouldn't have any l10n impact anyway. Mind putting these into variables the same way they're done in `apply.html`?

**Comment by mmmavis** [comment]
For sure! @craigcook I've updated my PR again. Let me know if I should squash my commits. :+1:

**Comment by craigcook** [comment]
r+ :hamster: Give 'em a squarsh and we're good to merge.

**Comment by mmmavis** [comment]
:bowtie: squashed!

**Comment by mmmavis** [comment]
yay :tada: thanks @craigcook

**Pull request merged**